js/everything.js

```diff
@@ -438,8 +453,15 @@ $(function() {
     // How do we know if the value is truly invalid?
     // Preload the form from the URL
     var currentVideoID = getCurrentVideoID();
+    var currentVideoPosition = getCurrentTimePosition();
+    if (currentVideoPosition) {
+        currentVideoPosition = "&t=" + currentVideoPosition;
+    }
+    else {
+        currentVideoPosition = "";
+    }
     if (currentVideoID) {
-        $("#v").attr("value", currentVideoID);
```

Comment:
You mean that the form should only contain the video ID in any case? So if user inputs i.e. `koJv-j1usoI&t=30` in the form, it should put the song at position 0:30 and display `koJv-j1usoI` ? I